src/components/autoHeightImage/autoHeightImage.tsx

```diff
@@ -39,11 +39,12 @@ export const AutoHeightImage = ({
   const imgStyle = {
     width: imgWidth - 10,
     height: imgHeight,
-    backgroundColor: onLoadCalled ? 'transparent' : EStyleSheet.value('$primaryGray'),
+    backgroundColor: onLoadCalled ? 'transparent' : EStyleSheet.value('$primaryLightBackground'),
```

Comment:
We have added image ratio on website, should consider adding that into mobile app as well and also start utilizing those ratio information.

Reply:
adding to backlog